### ISSUE TITLE
Add a material button to enable/disable the firewall

### ISSUE THREAD

**Issue by exadeci** [ukanth/afwall#402]
It would be a nice touch and useful to have that kind of switch on the bottom right

![fab](https://cloud.githubusercontent.com/assets/1394130/9112198/79ce655a-3c4d-11e5-857b-fd93d339ce87.png)

**Comment by CHEF-KOCH** [comment]
Thanks, but I'm against this future (in this form)  because the + normally indicates to 'add' something, e. g.  on WhatsApp it adds persons, creates chats/broadcasts or whatever.

We also have an ability to do this via the main menu or the widget. So my idea would be to leave it as it is or remove the entire menu and use such button instead to open the menu, but I see no improvement, clicking on the small menu or this button does the same, it's possible annoying on small phones.

**Comment by exadeci** [comment]
Well I was thinking a green with the logo and when you activate it goes red

**Comment by CHEF-KOCH** [comment]
Yes, that could be an indicator but without any text I see this problematic especially for beginners. Please note the other con arguments.

**Comment by CHEF-KOCH** [comment]
Or what also comes to my mind is that we could mod the notification icon option (default disabled under UI settings), after enabling this it shows if the firewall is running or not, since Android 4.x we got the ability to expand the notification area, swiping down shows/hides additional controls, I'm thinking about "Start" | "Stop" | "Settings" or something like this. Of course we would need some icons for each entry. 

Excuse my gfx _work_ :laughing:
![screenshot_2015-08-06-22-23-28](https://cloud.githubusercontent.com/assets/727123/9122711/1d6d07ba-3c8b-11e5-942d-b4ebb4672383.png)

**Comment by exadeci** [comment]
That would be really nice ! What about a profile switch on press for the "placeholder" ? 
Instead of my first suggestion you could add a navigation drawer with the list of profiles in it and other options from the current menu
 http://www.google.fr/design/spec/patterns/navigation-drawer.html#navigation-drawer-content

**Comment by CHEF-KOCH** [comment]
Jupp, that sounds very good to me. Thanks for the link, that's what I've looking for. :+1:

**Comment by Madis0** [comment]
A FAB wouldn't look good or relevant, it would be better to use a toggle on the top bar, but that just wouldn't fit between other buttons.  

I agree with the drawer idea.
